src/NUnitFramework/framework/Interfaces/TNode.cs

```diff
@@ -169,6 +169,13 @@ public static TNode FromXml(string xmlText)
                         }
                     }
                 }
+                else if (reader.NodeType == XmlNodeType.EndElement)
+                {
+                    if (reader.Depth < parents.Count)
+                    {
+                        parents.Pop();
```

Comment:
Does this mean that the code that 'Pops' in lines 153-157 can be removed?

Reply:
Good point! I recall that code being hit, but it was before having added my own changes. I'll try and check this out tonight to confirm if it can be removed.